### ISSUE TITLE
Cannot enable accessibility

### ISSUE THREAD

**Issue by hubbty** [KieronQuinn/TapTap#634]
On my device (Pixel 8a running GrapheneOS) Tap Tap and Tap Tap Secondary Service are greyed out and I can't enable them. I've got a log file. It does not contain the term "killed". So I do not know if this is a case of "service being killed by the system". 
See [https://notebin.de/?295d589802e5edda#5XjFka2fQCbLnE8VKAxQicPe5X42vLNUsu6sfxMy6Cxg](https://notebin.de/?295d589802e5edda#5XjFka2fQCbLnE8VKAxQicPe5X42vLNUsu6sfxMy6Cxg)

**Comment by KieronQuinn** [comment]
#582
